assignment/README.md

```diff
@@ -63,3 +104,6 @@ that you think would be useful? What technologies not covered seem exciting to
 you (you don't necessarily have to fully understand what they're for,
 this is a chance for you to get our help interpreting a technology's
 purpose/usage).
+
+
+Are there any ways to interact with csv (social explorer) data without using ArcGis?
```

Comment:
This SO post talks through reading a CSV in JS: https://stackoverflow.com/questions/7431268/how-to-read-data-from-csv-file-using-javascript

Alternatively, you can preprocess the CSV into JSON of the appropriate format (we did something much like this all the way back at the beginning of the semester)